vsts/pipelines/templates/_buildParallel.yml

```diff
@@ -1,15 +1,12 @@
 parameters:
```

Comment:
I feel this `_buildParallel.yml` is still monolithic having different tasks that we have in different build definitions...I think we should probably break this down more into reusable jobs?...something to think about...i might have missed few things...does you think this makes it better?

```
1. build, test and push build image (job type 1)

2. build, test and push runtime images (job type 2)

3. integration tests - <runtime-name> (job type 3)
   pull and retag build image
   pull and retag runtime images - <runtime-name>

4. signing job (job type 4)

validation.yml:
1. job type 1 (do not push)
2. job type 2 (do not push)

night.yml:
1. job type 1 (push)
2. job type 2 (push)
3. job type 3 (python)
4. job type 3 (php)
5. job type 3 (node)
6. job type 3 (dotnetcore)

ci.yml:
night.yml:
0. job type 4 (sign)
1. job type 1 (push)
2. job type 2 (push)
3. job type 3 (python)
4. job type 3 (php)
5. job type 3 (node)
6. job type 3 (dotnetcore)



```

Reply:
considering this as future work as there will be too many moveable parts